Library/Formula/terraform.rb

```diff
@@ -3,188 +3,85 @@
 
 class Terraform < Formula
   homepage "https://www.terraform.io/"
-  url "https://github.com/hashicorp/terraform/archive/v0.4.2.tar.gz"
-  sha1 "4d83a6db2879d277ffd662d1c0dcaad1da0e75e9"
+  url "https://github.com/hashicorp/terraform/archive/v0.5.0.tar.gz"
+  sha1 "6befb71e505efa246cb48e9ece829b93fea367fb"
 
   bottle do
     cellar :any
-    sha256 "ca67d033cf2900ffaa3deb25623dd73cbd6e126d7b78c0388c32703db84aa8b8" => :yosemite
-    sha256 "03e5765e051992610134cc4bf289c03ef6eb1a8355b25bda9544360aabd59f9e" => :mavericks
-    sha256 "893f7bcadd569adfc58e47acb72091daa4bd028cea770b0d638a8aee1488c7f5" => :mountain_lion
   end
 
   depends_on "go" => :build
 
-  go_resource "github.com/mitchellh/iochan" do
-    url "https://github.com/mitchellh/iochan.git", :revision => "b584a329b193e206025682ae6c10cdbe03b0cd77"
-  end
-
-  go_resource "github.com/mitchellh/gox" do
-    url "https://github.com/mitchellh/gox.git", :revision => "e8e6fd4fe12510cc46893dff18c5188a6a6dc549"
-  end
-
-  go_resource "golang.org/x/tools" do
-    url "https://go.googlesource.com/tools.git", :revision => "2a105dc4ba99792ac380f48d154af4e10b822846"
+  terraform_deps = %w[
+github.com/Sirupsen/logrus
+github.com/armon/circbuf
+github.com/awslabs/aws-sdk-go
+github.com/cyberdelia/heroku-go
+github.com/docker/docker
+github.com/dylanmei/iso8601
+github.com/dylanmei/winrmtest
+github.com/fsouza/go-dockerclient
+github.com/hashicorp/atlas-go
+github.com/hashicorp/aws-sdk-go
+github.com/hashicorp/consul
+github.com/hashicorp/errwrap
+github.com/hashicorp/go-checkpoint
+github.com/hashicorp/go-multierror
+github.com/hashicorp/go-version
+github.com/hashicorp/hcl
+github.com/hashicorp/yamux
+github.com/imdario/mergo
+github.com/masterzen/simplexml
+github.com/masterzen/winrm
+github.com/masterzen/xmlpath
+github.com/mitchellh/cli
+github.com/mitchellh/colorstring
+github.com/mitchellh/copystructure
+github.com/mitchellh/gox
+github.com/mitchellh/go-homedir
+github.com/mitchellh/iochan
+github.com/mitchellh/go-linereader
+github.com/mitchellh/mapstructure
+github.com/mitchellh/osext
+github.com/mitchellh/packer
+github.com/mitchellh/panicwrap
+github.com/mitchellh/prefixedio
+github.com/mitchellh/reflectwalk
+github.com/nu7hatch/gouuid
+github.com/packer-community/winrmcp
+github.com/pearkes/cloudflare
+github.com/pearkes/digitalocean
+github.com/pearkes/dnsimple
+github.com/pearkes/mailgun
+github.com/rackspace/gophercloud
+github.com/satori/go.uuid
+github.com/soniah/dnsmadeeasy
+github.com/vaughan0/go-ini
+github.com/xanzy/go-cloudstack
+  ]
+
+  terraform_deps.each do |x|
+    go_resource x do
+      url "https://#{x}.git"
```

Comment:
Go resources need a revision to make sure the formula doesn't break because of a dependency.